src/cmd-run-bwrap

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euo pipefail
+# Spawn the current build as a bwrap container.  Useful
+# for doing things like e.g. `rpm -q`.
+
+dn=$(dirname "$0")
+# shellcheck source=src/cmdlib.sh
+. "${dn}"/cmdlib.sh
+
+BUILDID=latest
+if ! [ -d "builds/${BUILDID}" ]; then
+    die "No builds/${BUILDID}"
+fi
+builddir=$(get_build_dir "${BUILDID}")
+
+commit=$(jq -r '.["ostree-commit"]' < "${builddir}/meta.json")
+
+tmproot=tmp/run-bwrap
+tmprootcommit=tmp/run-bwrap/.commit
+if ! [ -f "${tmprootcommit}" ] || ! [ "$(cat ${tmprootcommit})" = "${commit}" ]; then
+    sudo rm "${tmproot}" -rf
+    sudo ostree --repo=cache/repo-build checkout -UH "${commit}" "${tmproot}"
```

Comment:
Hmm, though in the unpriv path the build repo is already owned by the user. Let's make the `sudo` here conditional on`has_privileges`?

Reply:
This won't work in a !privileged scenario today like Kubernetes - and what we really want there anyways is to make a "exploded" oscontainer that we could e.g. schedule as a pod directly.